### ISSUE TITLE
LD icon missing at times

### ISSUE THREAD

**Issue by monte-hayward** [johnmichel/Library-Detector-for-Chrome#65]
The space just to the right of the red /F icon is blank.
Hover over it, and tool tip "Library Detector" is displayed.

When 1 or more libraries detected, their logos are shown.
So the Library Detector brand never gets seen.

![2015-04-25 at 9 44 22 am](https://cloud.githubusercontent.com/assets/3780422/7333309/32494c4a-eb30-11e4-8f2d-ce9dcbd65f50.png)

**Comment by Narno** [comment]
This is because if no libraries are detected, the icon is not hidden, it is just replaced by an empty image:
```php
function setNoIcon() {
    chrome.pageAction.setIcon({
        tabId: tabId,
        path: '../icons/null.png'
    });
    chrome.pageAction.setTitle({
        tabId: tabId,
        title: ''
    });
    chrome.pageAction.setPopup({
        'tabId': tabId,
        'popup': ''
    });
    chrome.pageAction.show(tabId);
}
```

**Comment by Narno** [comment]
May be it should use ```chrome.pageAction.hide(integer tabId)``` instead: https://developer.chrome.com/extensions/pageAction#method-hide

**Comment by Narno** [comment]
Hum, maybe there was a good reason to not to use "hide" function... @johnmichel? :sweat_smile:

**Comment by johnmichel** [comment]
Fixed with #67
